environment/cloudformation/training/IndyCluster.json

```diff
@@ -1393,7 +1393,7 @@
                     {
                       "Ref": "Validator01OperatingSystem"
                     },
-                    "indy"
+                    "sovrin"
```

Comment:
This is Indy repository. It should have no Sovrin dependencies.

Reply:
@ashcherbakov - This is just a key/name used to lookup a value in the RepositoryKeys hash here using Fn::FindInMap:

https://github.com/hyperledger/indy-node/blob/master/environment/cloudformation/training/IndyCluster.json#L5067

When you say this CloudFormation should not have Sovrin dependencies, are you referring to this repo URI?

https://github.com/hyperledger/indy-node/blob/master/environment/cloudformation/training/IndyCluster.json#L5078-L5079

A more detailed explanation is as follows:
The "68DB5E88" is appended to the following apt-key command when Fn::FindInMap is called on the RepositoryKeys map with the name of the operating system and the name of the repo:
`                "apt-key adv --keyserver keyserver.ubuntu.com --recv-keys ",
                {
                  "Fn::FindInMap": [
                    "RepositoryKeys",
                    {
                      "Ref": "Validator01OperatingSystem"
                    },
                    "indy"
                  ]
                },
                "\n",
`
The following add-apt-repository command adds debian repo "https://repo.sovrin.org/deb xenial" based on operating system and branch/codebase

`
                "add-apt-repository \"",
                {
                  "Fn::FindInMap": [
                    "RepositoryEvernym",
                    {
                      "Ref": "Validator01OperatingSystem"
                    },
                    {
                      "Ref": "CodeBase"
                    }
                  ]
                },`